packages/testing/config/jest/web/RedwoodWebJestEnv.js

```diff
@@ -10,5 +10,8 @@ module.exports = class RedwoodWebJestEnv extends TestEnvironment {
       this.global.TextEncoder = TextEncoder
       this.global.TextDecoder = TextDecoder
     }
+    if (typeof this.global.crypto.subtle === 'undefined') {
```

Comment:
Maybe add a comment here that this is required for Auth0?

Reply:
You right, thanks !